api.go

```diff
@@ -650,7 +650,7 @@ func handleAddOrUpdateApi(apiID string, r *http.Request) ([]byte, int) {
 		return createError("Marshalling failed"), 500
 	}
 
-	if err := ioutil.WriteFile(defFilePath, asByte, 0644); err != nil {
```

Comment:
`vetshadow` was suggesting these? IMO these changes should not be made. It makes these pieces of code depend on `err` being declared previously in the block.